services/inventory/agents.go

```diff
@@ -698,3 +698,54 @@ func (as *AgentsService) Remove(ctx context.Context, id string, force bool) erro
 
 	return nil
 }
+
+// AddRDSExporter inserts rds_exporter Agent with given parameters.
+func (as *AgentsService) AddRDSExporter(ctx context.Context, req *inventorypb.AddRDSExporterRequest) (*inventorypb.RDSExporter, error) {
+	var res *inventorypb.RDSExporter
+	e := as.db.InTransaction(func(tx *reform.TX) error {
+		params := &models.CreateAgentParams{
+			PMMAgentID:   req.PmmAgentId,
+			ServiceID:    req.ServiceId,
+			CustomLabels: req.CustomLabels,
+		}
+		row, err := models.CreateAgent(tx.Querier, models.RDSExporterType, params)
+		if err != nil {
+			return err
+		}
+		// TODO: do we have to check the RDS connection using the AWS API?
```

Comment:
Yes, we should check that passed access/secret keys work, or that credentials can be found by rds_exporter by other means.

Reply:
The check for other exporters is in `services/agents/registry.go` but there is no RDS type yet

Reply:
Let's keep that TODO for the separate ticket. We will need to add connection checker in pmm-agent for RDS.